reference/core/tls.md

```diff
@@ -138,8 +140,7 @@ spec:
 
 The TLS `Module` is deprecated. `TLSContext` should be used when using Ambassador version 0.50.0 and above.
 
-
-
+For users of the Ambassador Edge Stack, see the [Host CRD](/reference/host-crd) reference for more information.
```

Comment:
Shouldn't this be `../../reference/host-crd`?

Reply:
nope, we can go back to the regular style after GA